Book_apis/views.py

```diff
@@ -176,3 +178,92 @@ class BookDataAPIView(ListAPIView):
         'shop__name',
         'price'
     ]     
+
+
+class CreateStockAPIView(APIView):
+
+    permission_classes = [permissions.IsAuthenticated, IsVendorOnly]
```

Comment:
permission_classes = [permissions.IsAuthenticated, IsVendorOnly]

use tuple instead of list as we are not going to modify permission_classes in code after declare